core/templates/dev/head/pages/profile/Profile.js

```diff
@@ -126,14 +126,28 @@ oppia.controller('Profile', [
           $log.error('Error: cannot decrement page');
         } else {
           $scope.currentPageNumber--;
+          $scope.startingExplorationNumber = (
+            $scope.currentPageNumber * $scope.PAGE_SIZE + 1
+          );
+          $scope.endingExplorationNumber = (
+            Math.min($scope.numUserPortfolioExplorations,
```

Comment:
If you're going back one page, would you actually need Math.min()? I assume that page would be completely full...

Reply:
yeah.!
your right.!
removed it.
Thanks.